utils/uploadPictureToCloud.js

```diff
@@ -0,0 +1,22 @@
+const cloudinary = require('cloudinary').v2;
+const uniqid = require('uniqid');
+const fs = require('fs');
+
+const uploadPictureToCloudinary = async(files) => {
+
+    const photoPath = `./tmp/${uniqid()}.jpg`;
+    const resultMove = await files.mv(photoPath);
+
+    if(!resultMove) {
```

Comment:
Maybe you should add a comment here to explain why you're expecting a nullable value from .mv()